BonCodeAJP13/ServerPackets/BonCodeAJP13ForwardRequest.cs

```diff
@@ -566,10 +566,19 @@ private void WritePacket(NameValueCollection httpHeaders, String pathInfo, int s
                     }
                 }
             }
-            
+
             //WRITE NAMED ATTRIBUTES
 
 
+            //Check if the METHOD needs to be sent in the STORED METHOD Attribute IF SO we Set that Attribute and Value
+            if (method == BonCodeAJP13HTTPMethods.BONCODEAJP13_SC_M_JKSTORED)
+            {
+                string NonStandardVerb = GetKeyValue(httpHeaders, "REQUEST_METHOD");
+                pos = SetByte(aUserData, BonCodeAJP13HTTPAttributes.BONCODEAJP13_STORED_METHOD, pos); //attribute marker
+                pos = SetString(aUserData, NonStandardVerb, pos);  //method: e.g. we have clicked on URL
```

Comment:
line 578 - the example in comments should be probably something else, such `// override method: such as PATCH`

Reply:
How does this sound. 
//method to handle the passing of an arbitrary HTTP methods not handled by AJP 1.3 default binary methods

Reply:
Yes that will work.